.github/workflows/job-build-and-push-test-image.yml

```
@@ -0,0 +1,58 @@
+name: Build, Sign, Push and Tag test image
+
+on:
+  workflow_call:
+    inputs:
+      WORKING_DIRECTORY:
+        required: true
+        type: string
+    secrets:
+      GH_ACTIONS_ROLE_ARN:
+        required: true
+      TEST_IMAGE_REPOSITORY:
```

Suggestion only. Thought is that it then isn't ambiguous between whether this is the URI or ARN it wants. Same would go to the other refs in this file. Very minor happy for you to dismiss

```suggestion
      TEST_IMAGE_REPOSITORY_URI:
```

---

There is some alignment to do. Annoyingly the STS github secrets have a different naming convention. I'll action this comment

---

done